src/Http/ApiClient.php

```diff
@@ -39,15 +39,21 @@ class ApiClient
      * Error constants.
      */
     const ERROR_ENVIRONMENT_TYPE_UNKNOWN = 'Unknown environmentType "%s"';
-    const ERROR_MAC_OS_CURL_VERSION = 'Your PHP seems to be linked to the MacOS provided curl binary. ' .
-        'This is incompatible with our SDK, please reinstall by running: "brew reinstall %s --with-homebrew-curl".%s';
+    const ERROR_MAC_OS_CURL_VERSION = 'Your PHP seems to be linked to the MacOS provided curl binary.';
 
     /**
      * Public key locations.
      */
     const FILE_PUBLIC_KEY_ENVIRONMENT_SANDBOX = '/Certificate/sandbox.public.api.bunq.com.pubkey.pem';
     const FILE_PUBLIC_KEY_ENVIRONMENT_PRODUCTION = '/Certificate/api.bunq.com.pubkey.pem';
 
+    /**
+     * String format constants.
+     */
+    const FORMAT_CURL_INSTALLATION_INSTRUCTIONS =
+        'This is incompatible with our SDK, please reinstall by running: "brew reinstall %s --with-homebrew-curl".%s';
+    const FORMAT_ERROR_MESSAGE_MAC_CURL = '%s %s %s';
```

Comment:
Remove the spaces, otherwise the second line will start with a space, and there will be a useless trailing space after the first line as well.